fourkitchens/suhumsci
# shs-4955: fix ordering conflict between admin and toolbar css

## Summary
This fixes an issue in the shortcuts menu where two Drupal stylesheets were conflicting with one another.

## Steps to Test
1. Make sure you're logged in as someone with the `developer` role. For me, masquerade is broke, so I had to ULI in as Amit. 
2. With the Shortcuts menu active, hover over some of the parent menu items (Add Content, Manage Content, etc.).
3. The menu items should be vertical (top down) instead of horizontal (left right). Do this for a content page and a Drupal admin page as two stylesheets were needed.
